plugins/tiledb/io/TileDBWriter.cpp

```diff
@@ -347,7 +347,7 @@ void TileDBWriter::ready(pdal::BasePointTable &table)
             else
             {
                 // check attribute exists in original tiledb array
-                auto attrs = m_schema->attributes();
+                auto attrs = m_array->schema().attributes();
```

Comment:
I have reverted this back to `m_array->schema()` which was how it was in the original PR https://github.com/PDAL/PDAL/pull/2828 and this fixes a segfault in the test.